pymedphys/_dicom/anonymise.py

```diff
@@ -36,6 +37,18 @@
 
 HERE = dirname(abspath(__file__))
 
+logger = logging.getLogger(__name__)
+# basicConfig will only have effect if it was not called previously
+# initial testing indicated that it has not been invoked previously
+# however, future work to add logging to pymedphys and the CLI
+# in particular should precede and hence supercede this invocation
+logging.basicConfig(
+    filename="pymedphys_anonymise.log",
+    format="%(asctime)s|%(name)s|%(message)s",
+    datefmt="%Y/%m/%d %H:%M:%S",
+    level=logging.INFO,
+)
```

Comment:
Setting the logging config here will cause it to be adjusted on this module import (not when a particular function is run). This will result in awkward initiation logic.

Logging is currently already initialised within the CLI:

https://github.com/pymedphys/pymedphys/blob/cede62015203b09f9490e99c6eb6f653ed43c968/pymedphys/cli/__init__.py#L73

Could you use the same logging methods as used within https://github.com/pymedphys/pymedphys/blob/cede62015203b09f9490e99c6eb6f653ed43c968/pymedphys/_icom/listener.py#L24

And then, if you want to call the CLI with a debug log level you can run:

```
pymedphys --debug dicom anonymise
```